src/Components/Authentication/helpers.ts

```diff
@@ -1,3 +1,6 @@
+import * as sharify from "sharify"
```

Comment:
Interesting, didn't realize we had sharify in reaction too -- does this pick up data from force's config?

Reply:
I assumed so but now that I look for it, I'm not sure if it'd actually work once in Force. @xtina-starr is working on integrating `MobileSignUp` into Force so we'll see if it's working as expected.